### DISCUSSION TITLE
Having timing report written to .rpt file

### DISCUSSION THREAD

**Discussion by abaababa** [The-OpenROAD-Project/OpenROAD#3742] [Q&A]
Hello,

I am wondering if there is a way to generate .rpt files for timing report?
"report_worst_slack" directly shows the report in my terminal, but I want to record and compare different time slacks I get with different architecture for a design. 
I tried to use openroad -log to catch the reports but sometimes the log file is incomplete, is there any better way to do so?

Thanks

**Comment by maliberty** [comment]
In what way is the log incomplete?

**Reply by abaababa** [reply]
It might because I didn't exit openroad properly after the cmd_file was read. Not very sure.
I ran it with command "openroad <cmd_file> -log <log_file>",
And when the entire flow was done, I quitted using "ctrl+c". My log file ended with only part of the critical path listed and missed the rest path and the slack value.

I tried  "openroad <cmd_file> -log <log_file> -exit" and the log file is complete in that way.

**Reply by maliberty** [reply]
If you abort then it isn't surprising the file is incomplete.  I don't see a problem if you get a complete log when you exit normally.
